docs/changes.rst

```diff
@@ -15,6 +15,8 @@ Not yet released.
 * Improved automatic translation addon performance and reliability.
 * The rate limiting now can be customized in the Docker container.
 
+`All changes in detail <https://github.com/WeblateOrg/weblate/milestone/61?closed=1>`_.
```

Comment:
Use anonymous references here to avoid duplicate ones, see https://github.com/sphinx-doc/sphinx/issues/3921#issuecomment-315581557

Reply:
thank you for the hint